.gitmodules

```diff
@@ -37,6 +37,6 @@
 [submodule "libraries/boost"]
 	path = libraries/boost
 	url = https://github.com/boostorg/boost.git
-[submodule "libraries/libfc/libraries/boringssl/boringssl"]
-	path = libraries/libfc/libraries/boringssl/boringssl
-	url = https://github.com/AntelopeIO/boringssl-build
+[submodule "libraries/libfc/libraries/boringssl/bssl"]
+	path = libraries/libfc/libraries/boringssl/bssl
```

Comment:
Wondering why two levels `boringssl/bssl` in the path, not just `boringssl` or `bssl`?

Reply:
We still need our "wrapper" CMakeLists.txt that is in the `libraries/libfc/libraries/boringssl` directory, to then include upstream. All that could have just been placed further up in `libraries/libfc/libraries` but it's kinda long. In other words we still can't _just_ directly use the root of upstream's CMakeLists because we need to disable Werror, add install instructions, etc

I changed from `boringssl` to `bssl` to reduce grief when we switch between 1.0 & main because I think in the past we've had some annoyances with git in this regard... but maybe that was only when desubmoduling -- which this is not. So maybe unnecessary.